tools/build.gradle

```
@@ -16,4 +16,5 @@ dependencies {
 
   compile 'org.leadpony.justify:justify:3.1.0'
   compile 'org.glassfish:jakarta.json:2.0.1:module'
-}
\ No newline at end of file
+  implementation 'com.github.everit-org.json-schema:org.everit.json.schema:1.14.0'
```

```suggestion
  implementation 'com.github.erosb:everit-json-schema:1.14.0'
```
This variant seems to be the one available in Maven Central. This removes the need for configuring an extra repository.

---

The jitpack one is the official location, plut it's not a new repository, it's already included in https://github.com/openEHR/archie/blob/master/build.gradle#L42

However, this json-schema part is not technically part of the archie module build, so is not a submodule, but 'just' a gradle module using archie. That is why it is included here.

Makes me think maybe I should try to make it an Archie submodule again, which would remove the need for this kind of separate config. Or a separate repo just for this script?